### ISSUE TITLE
Unregister-PSFConfig leaves the config defined

### ISSUE THREAD

**Issue by nvarscar** [PowershellFrameworkCollective/psframework#175]
Maybe I'm doing something wrong, but I can't remove an existing config. I'm using `Unregister-PSFConfig -Module test -Name test`, but it still leaves the value for the config, whenever I open a new session.
How to repro:
```
Set-PSFConfig -Module test -Name test -Value test
Register-PSFConfig -Module test -Name test
#Check the value
Start powershell -ArgumentList '-noexit','-command &{ Get-PSFConfig -Module test }'
#Unregister the config
Unregister-PSFConfig -Module test -Name test
#Check the value - still there
Start powershell -ArgumentList '-noexit','-command &{ Get-PSFConfig -Module test }'
```
I'm on
```
Version    Name       
0.9.24.91  PSFramework
```
Am I doing something wrong here?

**Comment by FriedrichWeinmann** [comment]
Heya, no, you are _not_ doing something wrong here.
`PSFramework` is having a bug I haven't noticed yet, thanks for bringing it up.

Currently working on tests for the config system and going at it alphabetically, but this should ... not have happened to me, even without. Totally broke that in the last update, sorry :(

**Comment by FriedrichWeinmann** [comment]
I looked into the matter and found the issue - it's in how the command resolves configuration items with a Module/Name pair for input (where I screwed up royally).
I generally use the fullname on the `Register-PSFConfig` / `Unregister-PSFConfig` command set, so it didn't show during manual testing (making yet another point of how I need more automated tests).

As an interim solution you can instead call this:

```powershell
Unregister-PSFConfig -FullName test.test
```

But calling it with the more lenient filterset using `-Module` and `-Name` is definitely supported. Expect a fix soonest.

**Comment by FriedrichWeinmann** [comment]
Resolved in 0.9.24.98
